sdks/browser-sdk/CHANGELOG.md

```diff
@@ -1,5 +1,186 @@
 # @xmtp/browser-sdk
 
+## 2.0.0
+
+This release focuses on new features, stability, and performance.
+
+## Upgrade from 1.x to 2.0.0
+
+Use the information in these release notes to upgrade from `@xmtp/browser-sdk` 1.x to 2.0.0.
```

Comment:
Suggesting this based on v1.1.4 being the min SDK version required for May 1. Once the dev completes that upgrade - they can then focus on upgrading from 1.1.4 to 2.

```suggestion
Use the information in these release notes to upgrade from `@xmtp/browser-sdk` 1.1.4 to 2.0.0.
```